tests/Unit/ProtoTests/Basic/out/src/Client/BasicClient.php

```diff
@@ -208,4 +211,30 @@ public function methodWithArgs(RequestWithArgs $request, array $callOptions = []
     {
         return $this->startApiCall('MethodWithArgs', $request, $callOptions)->wait();
     }
+
+    /** Configure the gapic configuration to use a service emulator. */
+    private function setEmulatorConfig()
+    {
+        $emulatorHost = getenv('BASIC_EMULATOR_HOST');
+        if (!empty($emulatorHost)) {
+            if ($scheme = parse_url($emulatorHost, PHP_URL_SCHEME)) {
+                $search = $scheme . '://';
+                $emulatorHost = str_replace($search, '', $emulatorHost);
+            }
+
+            return [
+                'apiEndpoint' => $emulatorHost,
+                'transportConfig' => [
+                    'grpc' => [
+                        'stubOpts' => [
+                            'credentials' => ChannelCredentials::createInsecure(),
+                        ],
+                    ],
+                ],
+                'credentials' => new InsecureCredentials(),
```

Comment:
TODO: Change this to a working InsecureCredentialsWrapper. I could only find the wrapper in the Core!